libs/unity/library/Runtime/Scripts/Media/MediaSender.cs

```diff
@@ -126,6 +126,12 @@ public void Unmute()
         /// <inheritdoc/>
         protected async Task OnEnable()
         {
+#if PLATFORM_ANDROID
```

Comment:
If we #if out the whole `Android` class we then have to #if out all the places of call. We should #if out the method bodies instead, and possibly rename `Android` to `Platform` or something like this.

Reply:
I appreciate the first part, which is what Unity does I think; I guess you can `using Unity.Android` on Windows and it does nothing. On the second part I disagree because this is really Android-specific.

Reply:
Fixed.